include/net/xia_ether.h

```diff
@@ -0,0 +1,347 @@
+#ifndef _NET_XIA_ETHER_H
+#define _NET_XIA_ETHER_H
+/* prevents double declarations */
+
+#include <linux/netdevice.h>
+#include <linux/etherdevice.h>
+#include <linux/err.h>
+#include <net/xia_list_fib.h>
+#include <linux/netlink.h>
+#include <linux/spinlock.h>
+
+/* Ethernet Principal */
```

Comment:
Please add a period ``.`` at the end of comments. This applies to all the comments.